decidim-proposals/app/queries/decidim/proposals/metrics/accepted_proposals_metric_manage.rb

```diff
@@ -19,7 +19,7 @@ def query
           components = Decidim::Component.where(participatory_space: spaces).published
           @query = Decidim::Proposals::Proposal.where(component: components).joins(:component)
                                                .left_outer_joins(:category)
-          @query = @query.where("decidim_proposals_proposals.published_at <= ?", end_time).accepted
+          @query = @query.where("decidim_proposals_proposals.created_at <= ?", end_time).accepted
           @query = @query.group("decidim_categorizations.id", :participatory_space_type, :participatory_space_id)
           @query
         end
```

Comment:
You need to overwrite `ProposalMetricManage`'s `quantity` method to use `created_at` field, instead of `published_at`

Reply:
Thanks. The reason the test didn't catch it is because, in practice, an accepted proposal has to be *published*.

A *not published* proposal is a `draft`, a temporary proposal used in the proposal creation process that does not officially exist and **cannot** be accepted.